deploy/helm_charts/dc_website/README.md

```diff
@@ -0,0 +1,112 @@
+# Helm Charts
+
+The DC website can be installed into a GKE cluster using [helm](https://helm.sh/).
+
+## How to install the DC website on a live GKE cluster.
+
+### Prerequisites
+
+- A live GKE cluster with [workload identiy](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled. This is a one time setup.
+- [A global static IP reserved](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-external-ip-address).
+- Domain from your choice of DNS provider, and have the a [DNS type A record](https://en.wikipedia.org/wiki/List_of_DNS_record_types) which points from the domain to the IP from above.
+- Certificate for the domain provided. For example, from [Cloudflare](https://www.cloudflare.com/ssl/).
+
+Note: DNS record takes up to 72 hours to propagate. While the helm chart
+can still be installed before it is completed, the load balancer that is created by the chart may not be ready until then.
+
+### Installation
+
+1. Make sure that the k8s credentials are configured. You can visit the GCP UI and click "CONNECT" on your cluster's page to get the command to configure the credentials.
```

Comment:
does this require the cluster to be created already?

Reply:
Yes, Added some emphasis on cluster creation in the prerequisites